### DISCUSSION TITLE
About you and this bundle ?

### DISCUSSION THREAD

**Discussion by jvancoillie** [Guikingone/SchedulerBundle#140] [Show and tell]
I allow myself to take a little interest in your motivations, I know that it comes out a bit from the github discussions. 

What made you want to start developing this bundle?

What use do you make of it ? I'm curious to see the different uses that can be made of it, it can inspire me.

**Comment by Guikingone** [comment]
Hi @jvancoillie 👋🏻 

Sorry for the delay, I didn't receive the notification 🙁 

Well, long story short (that's a pretty long story), everything started back in 2020: https://github.com/symfony/symfony/pull/35315 when I opened a PR on Symfony. 

The idea was to offer something similar to Laravel `Schedule` in Symfony. 

I've discussed with Fabpot about this component but as for now, there's no plans to implement it into Symfony so lately, I've decided to open this bundle and work on it without waiting for Symfony 🙂 

Regarding my usage, I use it to schedule and execute repetitive tasks like consuming **Messenger** messages, emails and so on.

Thanks to the Probe and the HTTP entrypoint, I'm planning to use it to orchestrate projects at a higher level, in the near future, I've planned to work on a dedicated scheduler (maybe written in Rust) with a bridge via this bundle 🙂 

PS: Actually, the scheduler already exist but for personal reason, I've decided to not made it public yet 😉
